kvlds/btree_find.c

```diff
@@ -155,11 +160,13 @@ findleaf(void * cookie)
 
 		/* Fetch the child node.  This adds a lock to the parent. */
 		if (btree_node_fetch(C->T, C->N, findleaf, C))
-			goto err0;
+			goto err2;
 
 		/* Release the lock we acquired on the parent node. */
 		btree_node_unlock(C->T, NP);
 
+		/* Don't free C, because FIXME. */
```

Comment:
Empirically, I know that we can't free C here, but I don't understand why; quite likely as a result of my confusion about the recursive nature of `findleaf`.